features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtilsTest.kt

```diff
@@ -185,78 +227,86 @@ internal class DrawableUtilsTest {
         assertThat(width).isGreaterThanOrEqualTo(height)
 
         assertThat(displayMetricsCaptor.firstValue).isEqualTo(mockDisplayMetrics)
-
-        assertThat(result).isEqualTo(mockBitmap)
     }
 
+    // endregion
+
     @Test
-    fun `M return null W createBitmapFromDrawableOfApproxSize() { failed to create bmp }`() {
+    fun `M use bitmap from pool W createBitmapFromDrawable() { exists in pool }`(
```

Comment:
the method beign called is `createBitmapOfApproxSizeFromDrawable`, same for the tests below

```suggestion
    fun `M use bitmap from pool W createBitmapOfApproxSizeFromDrawable() { exists in pool }`(
```

Reply:
good catch - I missed that when renaming